codalab/worker/worker_run_state.py

```diff
@@ -461,6 +463,12 @@ def check_and_report_finished(run_state):
 
         def check_resource_utilization(run_state: RunState):
             logger.info(f'Checking resource utilization for bundle. uuid: {run_state.bundle.uuid}')
+            cpu_usage, memory_usage = docker_utils.get_container_stats_with_docker_stats(
+                run_state.container
+            )
+            run_state = run_state._replace(cpu_usage=cpu_usage)
```

Comment:
you should be able to just combine this into a single line

```suggestion
            run_state = run_state._replace(cpu_usage=cpu_usage, memory_usage=memory_usage)
```